astropy/utils/tests/test_data.py

```diff
@@ -2086,3 +2086,28 @@ def test_clear_download_cache_variants(temp_cache, valid_urls):
 def test_ftp_tls_auto(temp_cache):
     url = "ftp://anonymous:mail%40astropy.org@gdc.cddis.eosdis.nasa.gov/pub/products/iers/finals2000A.all"
     download_file(url)
+
+
+@pytest.mark.parametrize('base', ["http://exmaple.com", "https://example.com"])
```

Comment:
Is the typo intentional?
```suggestion
@pytest.mark.parametrize('base', ["http://example.com", "https://example.com"])
```